docs/site/decorators/Decorators_repository.md

```diff
@@ -214,7 +213,7 @@ Syntax:
 `@hasMany(targetResolver: EntityResolver<T>, definition?: Partial<HasManyDefinition>)`
 
 One-to-many connection between models e.g. a `TodoList` model has many of the
-`Todo` model. See [HasMany relation](HasMany-relation.md) for more details.
+`Todo` model. See [HasMany relation](HasMany-relation) for more details.
```

Comment:
Please change it to `../HasMany-relation.md`

Reply:
@raymondfeng 
Okay, Done with all changes.